examples/fabric-bridge-app/linux/main.cpp

```diff
@@ -74,7 +74,7 @@ void BridgePollingThread()
 #if defined(PW_RPC_FABRIC_BRIDGE_SERVICE) && PW_RPC_FABRIC_BRIDGE_SERVICE
             else if (ch == 'o')
             {
-                CHIP_ERROR err = OpenCommissioningWindow(0x1234);
+                CHIP_ERROR err = OpenCommissioningWindow(0x1234, 300, 3840, 1000, chip::NullOptional, chip::NullOptional);
```

Comment:
it is totally unreadable what 300 and 3840 and 1000 or the various NullOptional actually mean. Use param builders.

Reply:
Introduced separate param builder for example in https://github.com/project-chip/connectedhomeip/pull/34209/commits/d29e4fc464816478013110707686ad8d75a5d806